app/presenters/queries/content_item_presenter.rb

```diff
@@ -143,6 +145,11 @@ def parse_results(results)
             int_columns.each { |c| parse_int_column(result, c) }
             parse_state_history(result)
 
+            if self.include_warnings
```

Comment:
I've changed it to using the attr_accessor. Have a look at the code and tell me what you think? I'm unconvinced that there should be a method.